scripts/mods/mod_reforged/dynamic_spawns/units/undead_units.nut

```diff
@@ -24,8 +24,7 @@ local units = [
 	},
 	{
 		ID = "Unit.RF.SkeletonHeavyBodyguard",
-		Troop = "SkeletonHeavyBodyguard",
-		Figure = "figure_zombie_03"
```

Comment:
I think replace it with `figure_skeleton_03` because that's the honor guard figure. Also I have noticed that that figure is missing from `Unit.RF.SkeletonHeavy` so I'd say add that there too.